luigi/worker.py

```diff
@@ -1063,3 +1076,39 @@ def run(self):
             self._handle_next_task()
 
         return self.run_succeeded
+
+    def _handle_message(self, message):
+        logger.debug("Worker %s got message %s" % (self._id, message))
+
+        # the message is a tuple where the first element is an string that defines
+        # the function to call and the remaining elements are its arguments
+        name = message[0]
+        args = message[1:]
+
+        # find the function and check if it's callable and configured to work
+        # as a message callback
+        func = getattr(self, name, None)
+        tpl = (self._id, name)
+        if not callable(func):
+            logger.info("Worker %s has no function %s" % tpl)
+        elif not getattr(func, "is_message_callback", False):
+            logger.info("Worker %s function %s is not availale as message callback" % tpl)
+        else:
+            logger.info("Worker %s successfully dispatched message to function %s" % tpl)
+            func(*args)
+
+    @message_callback
+    def set_worker_processes(self, n, diff):
```

Comment:
Can't you make it into two different calls? One for `set_worker_processes_increment` and one for `set_worker_processes`. Both only taking one argument.

Or on second thought. I think it's better to not even having the `diff` version. I see these problems:

 * Not needed, humans trivially can make this calculation anyway.
 * If the UI is unresponsive humans are likely to set it twice, getting undesired effect.
 * Also, this is a new feature, let's start simple? :)

Altough, I'm also fine with including it. I don't have any strong opinions, I'll let you decide. :)